test/OpenTelemetry.AotCompatibility.TestApp/Program.cs

```diff
@@ -1,17 +1,4 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-using OpenTelemetry.AotCompatibility.TestApp;
-
-try
-{
-    PropertyFetcherAotTest.Test();
-}
-catch (Exception ex)
-{
-    Console.WriteLine(ex);
-    return -1;
-}
-
-Console.WriteLine("Passed.");
-return 0;
+Console.WriteLine("Hello, World!");
```

Comment:
Minor: any thoughts on why changing this from "Passed." to "Hello, world!"?

Reply:
I just switched files between this and contrib repository.

Reverted to passed in e0262317f86f304a4dc66a89abf2eeadf291adcc